mds/authent/migrations/0006_post_fill_allowed_scopes_with_default_values.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.7 on 2019-11-26 09:42
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+from django.db.models import F
+
+
+def fill_default_scopes(apps, schema_editor):
+    Application = apps.get_model("authent", "Application")
+
+    for application in Application.objects.all():
+        if application.users_allowed_scopes is None:
+            Application.objects.filter(users_allowed_scopes=None).update(
+                users_allowed_scopes=F("scopes")
+            )
```

Comment:
Wow wow wow, what happened? You loop on each application, and for each you batch update all the application objects?
```python
def fill_default_scopes(apps, schema_editor):
    Application = apps.get_model("authent", "Application")
    Application.objects.filter(users_allowed_scopes=None).update(
                users_allowed_scopes=F("scopes")
            )
```
does all you want. Have you checked what `F()` means?

Reply:
oups